scripts/check-public-api.ts

```diff
@@ -0,0 +1,359 @@
+/* eslint-disable jsdoc/require-jsdoc */
+
+import { join, resolve, parse, basename, dirname, sep, posix } from 'path';
+import path from 'path';
+import { fileURLToPath } from 'url';
+import { promises, existsSync } from 'fs';
+import { glob } from 'glob';
+import { createLogger, flatten, unixEOL } from '@sap-cloud-sdk/util';
+import mock from 'mock-fs';
+import { CompilerOptions } from 'typescript';
+import {
```

Comment:
Eslint is complaining about internal imports here, since it can't be prevented probably just exclude this line